Jenkinsfile-deploy

```diff
@@ -13,9 +13,14 @@ pipeline {
             }
         }
         steps {
-            sh 'sbt -no-colors test scalastyle assembly'
+            sh 'sbt -no-colors graphqlSchemaGen scalastyle assembly'
```

Comment:
Do we still want to run the `test` step?

Reply:
The assembly step runs the tests so we don't need it in there twice.